flipperdevices/flipperzero-firmware
# [FL-3363] More descriptive error messages for the log command

# What's new

- Inputting incorrect log level will give you the help message with a list of all the log levels
- Inputting a correct log level will remind you of how to list the log levels and begin outputting logs
- Inputting no log level will show what log level is set in settings and begin outputting logs

# Verification 

Using the Flipper CLI
- Run `log`
- Run `log debug`
- Run `log anything`
- Run `log ?` and/or `log help`

1 and 2 should show the log level and start outputting logs. 3 and 4 should show the help message and not output any logs.

# Checklist (For Reviewer)
